### DISCUSSION TITLE
As a developer using OSCAL 1.0.0rc1 I would like to understand what the update strategy is.

### DISCUSSION THREAD

**Discussion by butler54** [usnistgov/OSCAL#853] [Q&A] [answered]
Looking at the current website it appears that is is tracking the master branch in github, and not the tagged releases.

The way I have interpreted this is until we reach 1.0.0 I should be considering the schema in continuous flux. (with the rc's being capability milestones).

My assumption would be that when we hit 1.0.0 - the website would support a stable version as well as the current dev workstream.

Why am I asking? 
With https://github.com/IBM/compliance-trestle - we rely on automatic generation of code from the schema artefacts, however,
1) It's not perfect
2) Schema changes may influence our code, so we want to understand the expected frequency of updates
3) Partner developers rely on the schema webpages - so divergence becomes an issue.

**Comment by david-waltermire** [comment, marked as answer]
Yes. That's correct. We will continue to tag releases [as we have been](https://github.com/usnistgov/OSCAL/releases). These are referenced on the website under [releases](https://pages.nist.gov/OSCAL/contribute/roadmap/).

Starting with 1.0.0, we will start to maintain release and hotfix branches, perhaps using [this strategy](https://nvie.com/posts/a-successful-git-branching-model/).
